apps/atomy-q/API/tests/Feature/Api/V1/VendorGovernanceApiTest.php

```
@@ -0,0 +1,230 @@
+            'driver' => 'sqlite',
+            'database' => ':memory:',
+            'prefix' => '',
+            'foreign_key_constraints' => true,
+        ]);
+
+        return $app;
+    }
+
+    public function testGovernanceListsEvidenceFindingsAndScoresForTenantVendor(): void
+    {
+        Carbon::setTestNow(Carbon::parse('2026-04-22T00:00:00Z'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Carbon' in method 'testGovernanceListsEvidenceFindingsAndScoresForTenantVendor'. (undefined)

(StaticAccess)